src/util/dom_event.js

```diff
@@ -212,11 +212,17 @@
 
   function _getPointer(event, pageProp, clientProp) {
     var touchProp = event.type === 'touchend' ? 'changedTouches' : 'touches';
+    var pointer;
 
-    return (event[touchProp] && event[touchProp][0]
-      ? (event[touchProp][0][pageProp] - (event[touchProp][0][pageProp] - event[touchProp][0][clientProp]))
-        || event[clientProp]
-      : event[clientProp]);
+    if (event[touchProp] && event[touchProp][0]) {
+      pointer = event[touchProp][0][pageProp] - (event[touchProp][0][pageProp] - event[touchProp][0][clientProp]);
+    }
+
+    if (!pointer && pointer !== 0) {
```

Comment:
isn't this better as typeof pointer === 'undefined' ?  that is the real meaning of it, if yuo did manage to assign anything to pointer in the first if, right?

Reply:
yea it should work the same. I'll change it.